TROUBLESHOOTING.md

```diff
@@ -0,0 +1,420 @@
+* [compute.googleapis.com](#missing-api-computegoogleapiscom)
+* [iam.googleapis.com](#missing-api-iamgoogleapiscom)
+* [appengine.googleapis.com](#missing-appenginegoogleapiscom) (when managing an App Engine instance on the target project)
+
+#### Missing API: `cloudresourcemanager.googleapis.com`
+
+**Error message:**
+
+```
+Error: Error refreshing state: 1 error(s) occurred:
+
+* module.project-factory.data.google_organization.org: 1 error(s) occurred:
+
+* module.project-factory.data.google_organization.org: data.google_organization.org: Error reading Organization Not Found : [organization-id]: googleapi: Error 403: Cloud Resource Manager API has not been used in project [seed-project-number] before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=[seed-project-number] then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry., accessNotConfigured
+
+```
+
+**Cause:**
+
+The seed project does not have the `cloudresourcemanager.googleapis.com` API enabled. This prevents the Project Factory from looking up the GCP organization.
+
+**Solution:**
+
+```
+# Requires `roles/serviceusage.serviceUsageAdmin` on $SEED_PROJECT
+gcloud services enable cloudresourcemanager.googleapis.com --project "$SEED_PROJECT"
+```
+
+#### Missing API: `cloudbilling.googleapis.com`
+
+**Error message:**
+
+```
+Error: Error applying plan:
+
+1 error(s) occurred:
+
+* module.project-factory.google_project.project: 1 error(s) occurred:
+
+* google_project.project: Error setting billing account "[billing-account]" for project "projects/[target-project-id]": googleapi: Error 403: Cloud Billing API has not been used in project [seed-project-number] before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudbilling.googleapis.com/overview?project=[seed-project-number] then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry., accessNotConfigured
+```
+
+**Cause:**
+
+The seed project does not have the `cloudbilling.googleapis.com` API enabled. This prevents the Project Factory from enabling APIs that may result in billing charges, such as the Google Compute Engine API.
+
+**Solution:**
+
+```
+# Requires `roles/serviceusage.serviceUsageAdmin` on $SEED_PROJECT
+$ gcloud services enable cloudbilling.googleapis.com --project "$SEED_PROJECT"
+```
+
+**Notes:**
+
+This error will occur *once* when applying a Terraform plan when the **seed project** does not have the Cloud Billing API enabled. On subsequent Terraform runs, Terraform will generate an error indicating that the Compute Engine API is not enabled on the **target project**. Watch out for this!
+
+### Missing API: `compute.googleapis.com`
+
+TODO: it appears that the `compute` API isn't actually used by the project factory. Can we confirm this?
```

Comment:
I believe we use it to destroy the default network? I'd prefer to not remove it from the list but we can remove the Troubleshooting section.

Reply:
I was surprised to find that the README doesn't state `compute.googleapis.com` as an API - I always assumed it was there. Will delete this section from the troubleshooting guide.